app/views/manuals/_manual_updates.html.erb

```diff
@@ -1,12 +1,11 @@
 <div class='manual-body' id="content">
   <article aria-labelledby="section-title">
     <%= render "govuk_publishing_components/components/heading", {
-      text: "Updates: #{presented_manual.title}",
       font_size: "l",
       heading_level: 1,
       id: "section-title",
+      text: "Updates: #{presented_manual.title}",
```

Comment:
I feel like `margin_bottom: 3` or something would be apt here to maintain the spacing:

![Screenshot 2021-03-04 at 18 02 12](https://user-images.githubusercontent.com/71266765/110008649-dc5f0f00-7d13-11eb-8cdc-09f544260a7c.png)

Reply:
Done!